shared/actions/fs/common.native.tsx

```diff
@@ -11,11 +11,19 @@ import {saveAttachmentDialog, showShareActionSheetFromURL} from '../platform-spe
 import {types} from '@babel/core'
 
 const pickAndUploadToPromise = (state: TypedState, action: FsGen.PickAndUploadPayload): Promise<any> =>
-  new Promise((resolve, reject) =>
-    ImagePicker.showImagePicker(
+  new Promise((resolve, reject) => {
+    let title = 'Select a Photo'
+    let captureTitle = 'Take Photo...'
+    if (action.payload.type === Types.MobilePickType.Video) {
+      title = 'Select a Video'
+      captureTitle = 'Take Video...'
+    }
```

Comment:
On iOS we used `Mixed`. Not sure if the title is shown there though. If shown, we should probably use a switch/case here and say something like "Select a photo or video". If not shown, I'd prefer not setting them on iOS in case they start to show up at some point in the future.